index.html

```diff
@@ -2598,9 +2598,11 @@ <h2 id="View">Backbone.View</h2>
       whether they've already been inserted into the page or not. In this
       fashion, views can be rendered at any time, and inserted into the DOM all
       at once, in order to get high-performance UI rendering with as few
-      reflows and repaints as possible. <tt>this.el</tt> is created from the
-      view's <tt>tagName</tt>, <tt>className</tt>, <tt>id</tt> and <tt>attributes</tt> properties,
-      if specified. If not, <b>el</b> is an empty <tt>div</tt>.
+      reflows and repaints as possible. <tt>this.el</tt> is either a DOM selector
+      string or Element, or created from the view's <tt>tagName</tt>, <tt>className</tt>,
+      <tt>id</tt> and <tt>attributes</tt> properties, if specified. If not,
```

Comment:
Can you link these properties where possible? 

http://backbonejs.org/#View-attributes

also that comma there is sort of awkward. I would word this

> `this.el` can be resolved from a DOM selector string or an Element; otherwise it will be created from the view's `tagName`, `className`, `id` and `attributes` properties specified.